custom/icds_reports/utils/aggregation_helpers/distributed/child_health_monthly.py

```diff
@@ -93,8 +94,12 @@ def _state_aggregation_query(self, state_id):
         )
         migration_status = "(agg_migration.is_migrated=1 AND agg_migration.migration_date < {})::integer".format(
             start_month_string)
-        seeking_services = "(person_cases.registered_status IS DISTINCT FROM 0 AND {} IS DISTINCT FROM 1)".format(
-            migration_status)
+        availing_services = (
+            "CASE WHEN NOT (agg_as.is_registered=0 AND agg_as.registration_date < {})"
+            "THEN 1 ELSE 0 END"
+        ).format(start_month_string)
+        seeking_services = "(person_cases.registered_status IS DISTINCT FROM 0 AND {} IS DISTINCT FROM 1 AND {} IS DISTINCT FROM 0)".format(
```

Comment:
what is the distinction here between seeking services and availing services? we use those phrases interchangeably in a lot of places so its very confusing to have them be distinct here